README.md

```diff
@@ -35,6 +35,35 @@ gf("Stawka wieksza niz zycie", function(err, links) {
 });
   ```
 
+// Get all hosts
+// Available files type : Archives | Audio | Video | Docs | Pictures | Applications | CD,DVD
+```javascript
+
+getAllHosts(function(err, data){
+	console.log(data);
+    //[ { key: '500', name: 'direct links' },
+    //  { key: '78', name: 'torrents' },
+    //  { key: '23', name: '4shared.com' },
+    // ... ]
+});
+
+  ```
+  
+// Filter by type of file
+// Available files type : Archives | Audio | Video | Docs | Pictures | Applications | CD,DVD
+```javascript
+
+var options = {types: Video };
```

Comment:
I believe those should be strings, right? Like:
```
var options = {types: 'Video'}
```
Also - what do you think about changing the name of the option from `types` to singular `type` ?